content/guides/yarn.md

```diff
@@ -0,0 +1,51 @@
++++
```

Comment:
This is going to be a little harder than this article suggests. To speed builds up, the package.json is md5'ed and this is checked on build to see if there has been any updates. I think we'll need to put this on hold for a bit.